server/utils/sendEmail.js

```diff
@@ -4,20 +4,19 @@
 
 const sgMail = require("@sendgrid/mail");
 
-const sendEmail = async () => {
+const sendEmail = async (toEmail, { subject, html, text }) => {
   sgMail.setApiKey(process.env.SENDGRID_API_KEY);
 
   const msg = {
-    to: "amahalwy@gmail.com", // Change to your recipient
-    from: "teamkanban.hatchways@gmail.com", // Change to your verified sender
-    subject: "Sending with SendGrid is Fun",
-    text: "and easy to do anywhere, even with Node.js",
-    html: "<strong>and easy to do anywhere, even with Node.js</strong>",
-    templateId: "d-e6c0b94a01e1459c9d4927595a785b97",
+    to: toEmail,
+    from: 'ethanmoffat@hey.com',
+    subject: subject,
```

Comment:
The `from` email usually shouldn't be the same as your personal email but I did this for testing purposes. This should be updated in a configuration file with a verified sender that you setup in your SendGrid.